bundle-size/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "dev": "nodemon",
-    "deploy": "gcloud app deploy",
+    "deploy": "gcloud config set project amp-bundle-size-bot && gcloud app deploy",
```

Comment:
This will silently change the global default if you've set it to something else. Instead, I suggest using `--project`, which sets the project just for this invocation.

https://cloud.google.com/sdk/gcloud/reference/#--project

Reply:
Done